src/Emogrifier/HtmlProcessor/HtmlPruner.php

```diff
@@ -41,4 +44,69 @@ public function removeInvisibleNodes()
 
         return $this;
     }
+
+    /**
+     * Removes classes that are no longer required (e.g. because there are no longer any CSS rules that reference them)
```

Comment:
Should there be a comma after "e.g."?

Reply:
There's some discussion on [StackExchange](https://english.stackexchange.com/questions/16172/should-i-always-use-a-comma-after-e-g-or-i-e) that suggests it may be a matter of style, but also an [answer with an example in brackets](https://english.stackexchange.com/questions/16172/should-i-always-use-a-comma-after-e-g-or-i-e#answer-16188) (as here) where there is no comma, and also a [reference link showing a comma only when the example is not in brackets](https://theoatmeal.com/comics/ie).  So I'll leave it as it is.